sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/context/AzureTokenCredentialAutoConfiguration.java

```diff
@@ -101,6 +104,11 @@ AzureTokenCredentialResolver azureTokenCredentialResolver(
                 return null;
             }
 
+            String tokenCredentialBeanName = azureProperties.getCredential().getTokenCredentialBeanName();
```

Comment:
Maybe we could reconsider this later, do we need two beans referencing the same object here?

Reply:
Yes, will continue to improve